semantic error: unhashable type: 'list'
 --> tests/errors/test_set_object1.py:4:31
  |
4 |     my_set: set[list[i32]] = {[1, 2], [3, 4]}
  |                               ^^^^^^ 
